proto/buf.lock

```diff
@@ -4,8 +4,10 @@ deps:
   - remote: buf.build
     owner: googleapis
     repository: googleapis
-    commit: 75b4300737fb4efca0831636be94e517
+    commit: cc916c31859748a68fd229a3c8d7a2e8
```

Comment:
The latest commit for googleapis on buf.build is [`cc916c31859748a68fd229a3c8d7a2e8`](https://buf.build/googleapis/googleapis/commits/cc916c31859748a68fd229a3c8d7a2e8)

https://buf.build/googleapis/googleapis/commits/main